observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,63 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="^1,^1,^1,^1,^1,^1", options="header"]
+|===
+| *COO Version*   | *OCP Versions*       | *Dashboards*   | *Distributed Tracing*   | *Logging*   | *Troubleshooting Panel* 
+
+| 0.2.0           | 4.11                 | ✔              | ✘                       | ✘           | ✘ 
+| 0.3.0+          | 4.11 - 4.15          | ✔              | ✔                       | ✔           | ✘ 
+| 0.3.0+          | 4.16+                | ✔              | ✔                       | ✔           | ✔ 
+|===
+
+
+[id="cluster-observability-operator-release-notes-0-4-0_{context}"]
+== {coo-full} 0.4.0
+
+The following advisory is available for {coo-full} 0.4.0:
+
+// Change the errata link after release and just before publishing
+* link://https://errata.devel.redhat.com/advisory/138050[138050 {coo-full} 0.4.0]
+// * link:https://access.redhat.com/errata/RHEA-2024:????[RHEA-2024:??? {coo-full} 0.4.0]
+
+
+[id="cluster-observability-operator-0-4-0-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+// COO-264
+==== Troubleshooting UI plugin
+
+* The troubleshooting UI panel has been improved so you can now select and focus on a specific starting signal.
+* There is more visibility into Korrel8r queries, with the option of selecting the depth.
+* Users of {product-title} version 4.17+ can access the troubleshooting UI panel from the Application Launcher {launch}. Alternatively, on versions 4.16+, you can access it in the web console by clicking on **Observe** -> **Alerting**.   
+
+//
+// For more information, see xref :../../observability/cluster_observability_operator/ui_plugins/troubleshooting-ui-plugin.adoc#troubleshooting-ui-plugin[troubleshooting UI plugin]
+
+// COO-263
+==== Distributed tracing UI plugin
+
+* The distributed tracing UI plugin has been enhanced, with a Gannt chart now avilable for exploring traces.
+//
+// For more information, see xref :../../observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc#distributed-tracing-ui-plugin[distributed tracing UI plugin]
```

Comment:
I assume the fact that this line is commented out is the final intended state.

Reply:
I am planning to merge the content that is the target to for the xref after the rel notes are merged - so I can update these commented out links then